assets/js/swedbank-pay-design-guide-theme.js

```diff
@@ -11,40 +12,41 @@
         flowchart: {
             useMaxWidth: false
         }
-    });
+    }
+    mermaid.initialize(configObject);
 
-    mermaid.init(undefined, 'code.language-mermaid');
+    mermaid.init(configObject, "code.language-mermaid");

```

Comment:
I think `init` takes an HTML element, not configuration. The configuration is sent to `initialize()`.